outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
I agree that the motivation should be short (narrative) but also give an introduction to the use case list we already generated. I think the use case list could be displayed in a table: one axis organized by stakeholder (researcher, publisher, librarian, etc.) and the other axis by activity/need. I believe that is essentially how that list is already organized.